dependabot/status.py

```diff
@@ -0,0 +1,142 @@
+import ci.util
+import datetime
+import github3.repos
+import tabulate
+import typing
+
+from dependabot.model import (
+    DependebotStatus,
+    DependabotStatusForRepo,
+)
+import dependabot.util
+
+
+def _retrieve_known_dependabot_file(
+    repository: github3.repos.repo.ShortRepository,
+):
+
+    # According to https://docs.github.com/en/github/administering-a-repository/configuration-options
+    # -for-dependency-updates , there is only one valid path for the dependabot.yml.
+    try:
+        file = repository.file_contents(path='.github/dependabot.yml')
+        ci.util.info(f'dependabot in {repository.full_name}')
+        return file
+    except github3.exceptions.NotFoundError:
+        ci.util.info(f'dependabot not in {repository.full_name}')
+        return None
+
+
+def _validate_dependabot_file(
+    dependabot_file,
+) -> bool:
+
+    # TODO to be extended, e.g. scan for required attributes, see:
+    # https://docs.github.com/en/github/administering-a-repository/
+    # configuration-options-for-dependency-updates
+    content = dependabot_file.decoded.decode("utf-8")
```

Comment:
nit: by convention, we tend to prefer to use single-quotes in cc-utils

Reply:
Absolutely.
Maybe we should extend our linter to check for unnecessary double quotation 🤔?

see: https://github.com/zheller/flake8-quotes